frontend/.env

```diff
@@ -1,3 +1,3 @@
 SKIP_PREFLIGHT_CHECK=true
 BROWSER=none
-REACT_APP_API_URL=http://localhost/api/v1
+REACT_APP_API_URL=/api/v1
```

Comment:
Will this work in local development? The frontend is running on a different port in that case, right? @nikita-vostrosablin. If it doesn't work, you should be able to use a `.env.development` file containing `REACT_APP_API_URL=http://localhost/api/v1`... it will be picked up by `create-react-app`